src/SFA.DAS.FindEpao.Web.UnitTests/Controllers/CoursesControllerTests/WhenGettingCourseEpao.cs

```diff
@@ -88,6 +88,7 @@ public class WhenGettingCourseEpao
             model!.EffectiveFrom.Should().Be(mediatorResult.EffectiveFrom);
             model!.AllCourses.Should().BeEquivalentTo(
                 mediatorResult.AllCourses.Select(item => (CourseListItemViewModel)item));
+
```

Comment:
Do you need a Standard Version assert here?

Reply:
updated